### PR TITLE
Add openapi.tanna.dev/go/validator

### PR DESCRIPTION
**Pull request by jamietanna**

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1218,6 +1218,14 @@
   description: OpenAPI 3.0 (and Swagger v2) implementation for Go (parsing, converting, validation, and more)
   v3: true
 
+- name: openapi.tanna.dev/go/validator
+  category:
+    - data-validators
+  github: https://gitlab.com/jamietanna/httptest-openapi/
```
Comment (jamietanna) [inline _data/tools.yml:1224]: Even though this isn't on GitHub, this should be fine, right?

Comment (philsturgeon) [reply _data/tools.yml:1224]: We should change this to source code. @mbifulco can you consider this for the rewrite?

**Review by philsturgeon — APPROVED** [review]

**Comment by jamietanna** [comment]
Manually merged with https://github.com/apisyouwonthate/openapi.tools/commit/eba8ea9b5b18272f9594061b46f9ac477f930431

File: _data/tools.yml (hunk 1 above)
Comment (mbifulco) [reply _data/tools.yml]: I'll change it to `repo:` !

**Pull request closed without merging**